components/VotesList/VotesListItem.tsx

```diff
@@ -273,18 +293,57 @@ export function VotesTableRow({
   );
 }
 
+function VoteText({ voteText }: { voteText: string | undefined }) {
```

Comment:
If a vote is very long we truncate it and add a tooltip that shows the full value